vspec/model/vsstree.py

```diff
@@ -42,6 +42,7 @@ class VSSNode(Node):
     min = ""
     max = ""
     allowed = ""
+    instantiate = True
```

Comment:
A lot of programmers prefix booleans by convention either with "is" or "has" like for example `user.isAdmin`. This can be helpful to identify a var as boolean immediately when reading the name. It can help too with the autocomplete feature of the IDE. Following this standard the value here would normally be something like `isInstance`.

Reply:
@tobiask84, valid point. Just thinking that `isInstance` is not correct, as it's not an instance yet but will be instantiated. Then to be consequent we would have to change `allow` to `isAllowed`, `max` to `hasMax` etc. I don't know if we wanna go there.

But don't get me wrong, I'm all in for making it easier to understand.